crates/jstzd/tests/octez_client_test.rs

```diff
@@ -61,6 +61,55 @@ async fn generates_keys() {
     assert_eq!(secret_keys["name"], alias);
 }
 
+#[tokio::test]
+async fn show_address() {
+    let temp_dir = TempDir::new().unwrap();
+    let base_dir = temp_dir.path().to_path_buf();
+    let octez_client = OctezClientBuilder::new()
+        .set_base_dir(base_dir.clone())
+        .build()
+        .unwrap();
+    let alias = "test_alias".to_string();
+    let _ = octez_client.gen_keys(&alias, None).await;
+    let res = octez_client.show_address(&alias, false).await;
+
+    assert!(res.is_ok_and(|addr| {
+        addr.hash.to_string().starts_with("tz1")
+            && addr.public_key.to_string().starts_with("edpk")
+    }));
+}
+
+#[tokio::test]
+async fn show_address_with_secret_key() {
+    let temp_dir = TempDir::new().unwrap();
+    let base_dir = temp_dir.path().to_path_buf();
+    let octez_client = OctezClientBuilder::new()
+        .set_base_dir(base_dir.clone())
+        .build()
+        .unwrap();
+    let alias = "test_alias".to_string();
+    let _ = octez_client.gen_keys(&alias, None).await;
+    let res = octez_client.show_address(&alias, true).await;
+    println!("{:?}", res);
```

Comment:
```suggestion
```

Reply:
good catch, thanks 🙏

Reply:
It's still here though and there's a new one in `address_try_from` :)